test/integration/acceptance/query_permission_test_base.hpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TEST_INTEGRATION_ACCEPTANCE_QUERY_PERMISSION_TEST_BASE_HPP_H_
+#define IROHA_TEST_INTEGRATION_ACCEPTANCE_QUERY_PERMISSION_TEST_BASE_HPP_H_
+
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "framework/specified_visitor.hpp"
+#include "integration/acceptance/query_permission_fixture.hpp"
+#include "utils/query_error_response_visitor.hpp"
+
+using namespace shared_model;
+using namespace integration_framework;
+using namespace shared_model::interface::permissions;
```

Comment:
Though we usually against the `using namespace` in any header that one probably is ok. Would like to have others comments

Reply:
My vote is for avoidance namespaces usage in headers

Reply:
removed these from headers.